eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/OfflineStateSpec.scala

```diff
@@ -492,7 +488,7 @@ class OfflineStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     alice2blockchain.expectMsg(PublishAsap(aliceCommitTx))
   }
 
-  test("handle feerate changes while offline (don't close on mismatch)", Tag("disable-offline-mismatch")) { f =>
+  test("handle feerate changes while offline (don't close on mismatch, update on reconnect)", Tag("disable-offline-mismatch")) { f =>
```

Comment:
The test is good, but you have kind of hacked an existing unrelated one. I think this should be a separate test, a bit like "re-send channel_update at reconnection for private channels".

Reply:
Good point, done.